common/src/loki_logger.cpp

```diff
@@ -0,0 +1,76 @@
+#include "loki_logger.h"
+
+// clang-format off
```

Comment:
what is this for?

Reply:
It's to prevent `clang-format` from re-ordering the two `#includes` on the following lines, which breaks the build somehow.

```
error: no member named 'null_mutex' in namespace 'spdlog::details'
error: no template named 'rotating_file_sink_st' in namespace 'spdlog::sinks'; did you
      mean 'rotating_file_sink'?
```

Reply:
Ah, yes, forgot that.